velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -1102,6 +1102,21 @@ class ArrowBridgeArrayImportTest : public ArrowBridgeArrayExportTest {
     }
   }
 
+  void testShortDecimalImport() {
```

Comment:
Could we reuse the method `testArrowImport`? Maybe a template parameter TOuput and a default argument for the output values need to be added.

Reply:
Hey, @rui-mo I tried to reuse the method but the codes looks lengthy and not elegant... Could you pls give me more hint in case I miss something We have to pass `int128_t` as arrowInput while `int64_t` as vector raw values, and take these values as `Decimal` type...😣

Reply:
I implemented some code in https://github.com/rui-mo/velox/commit/a2d33ffadc98617a47f966fa3a9a7170119e1cd1. Please check if it makes sense, thanks.